packages/protocol/migrations/20_elect_validators.ts

```diff
@@ -117,13 +117,15 @@ async function registerValidator(
   ).toString('hex')
   const publicKeysData = publicKey + blsPublicKey + blsPoP
 
+  console.log('locking gold for validator registration')
```

Comment:
Mega-nit: I think by convention we capitalize these debug logs in migrations. Here and below.

Reply:
Removed, had added for debugging